tests/tensor_search/integ_tests/test_embed.py

```diff
@@ -150,6 +151,45 @@ def tearDown(self) -> None:
         super().tearDown()
         self.device_patcher.stop()
 
+    def test_embed_content_cuda_device_as_default(self):
+        """
+        Test that embed_content uses the default device when no device is specified.
+        """
+        for index in [self.unstructured_default_text_index, self.structured_default_text_index]:
+            with self.subTest(index=index.type):
+                expected_device = "cuda"
```

Comment:
You should test cpu too. This would pass if you were always using cuda regardless of this config

Reply:
added cpu as test